exec/java-exec/src/main/java/org/apache/drill/exec/compile/ClassBuilder.java

```diff
@@ -141,7 +142,9 @@ public ClassBuilder(DrillConfig config, OptionManager optionManager) {
     // A key advantage of this method is that the code can be
     // saved and debugged, if needed.
 
-    saveCode(code, name);
+    if (cg.persistCode()) {
```

Comment:
It is the method to ask that the generated code be saved... Perhaps the confusing bit is that there are two methods of the same name: `persistCode(boolean)` which is called to ask to persist the code, and `persistCode()` which is used to check if the code should be persisted. (Somehow, `isPersistCode` or `isCodePersisted` just didn't seem as clear...)

On the next PR, I'll see if I can come up with a clearer name. At that time, I'll find a way to test the two "dead" operators so that all support plain Java. Once that is done, I can remove the calls to `supportsPlainJava` and comments about using `persistCode` that now appear in each operator to mark those that are plain-Java ready.